src/objects/list/list.scss

```diff
@@ -0,0 +1,41 @@
+@use '../../design-tokens/sizes.yml';
+
+/**
+ * Override browser defaults. Note that this means it's very important to
+ * include `[role="list"]` to prevent disruption of list navigation in Safari
+ * VoiceOver.
```

Comment:
> in Safari VoiceOver.

Question: Does this only occur when using Safari + VoiceOver? I read the reference link but it was not clear to me whether this occurs when using Chrome + VoiceOver.

Reply:
Only Safari.